src/react-native/20-building/building.md

```diff
@@ -12,37 +12,53 @@ In this section, you will:
 
 - TODO
 
-## Objective 1: TODO
+## Objective 1: Creating a build
 
 TODO
 
 ### Concept TODO
 
-TODO
+<!-- Link to provide steps for signing and building -->
 
-### Setup 1
+https://reactnative.dev/docs/signed-apk-android
 
-TODO
+### Setup 1
 
-### Verify 1
+✏️ Update **package.json** to be:
 
-TODO
+@diff ../../../exercises/react-native/19-performance/01-solution/package.json ../../../exercises/react-native/20-building/01-solution/package.json only
 
 ### Exercise 1
 
-TODO
+✏️ Run to generate AAB file:
+
+```bash
+npm run build
+```
 
 ### Solution 1
 
-If you’ve implemented the solution correctly, the tests will pass when you run `npm run test`!
+If there is a successful build, the file **android/app/build/outputs/bundle/release/app-release.aab** will be created.
```

Comment:
What happens if there isn’t a successful build? 😅

Reply:
Also, can the AAB be run in the emulator? If so, we should instruct them to do that.